### PR TITLE
[type:refactor #3088]format auth-param-sqlmap.xml 

### PR DESCRIPTION
**Pull request by hutaishi**
format auth-param-sqlmap.xml #3088

### CODE REVIEW

File: shenyu-admin/src/main/resources/mappers/auth-param-sqlmap.xml [hunk 1]
```diff
@@ -28,100 +28,103 @@
...
 
     <select id="findByAuthId" parameterType="java.lang.String" resultMap="BaseResultMap">
-        select
-        <include refid="Base_Column_List"/>
-        from auth_param
-        where auth_id = #{authId,jdbcType=VARCHAR}
+        SELECT
+            <include refid="Base_Column_List"/>
+        FROM auth_param
+        WHERE auth_id = #{authId, jdbcType=VARCHAR}
```
Comment (midnight2104) [inline shenyu-admin/src/main/resources/mappers/auth-param-sqlmap.xml:43]: The keywords are not right-aligned.

File: shenyu-admin/src/main/resources/mappers/auth-param-sqlmap.xml [hunk 2]
```diff
@@ -28,100 +28,103 @@
...
-        <include refid="Base_Column_List"/>
-        from auth_param
-        where auth_id in
-        <foreach collection="authIdSet" index="index" item="authId"
-                 open="(" separator="," close=")" >
-            #{authId,jdbcType=VARCHAR}
-        </foreach>
+        SELECT
+            <include refid="Base_Column_List"/>
+        FROM auth_param
```
Comment (midnight2104) [inline shenyu-admin/src/main/resources/mappers/auth-param-sqlmap.xml:49]: The keywords are not right-aligned.

File: shenyu-admin/src/main/resources/mappers/auth-param-sqlmap.xml [hunk 3]
```diff
@@ -28,100 +28,103 @@
...
-        select
-        <include refid="Base_Column_List"/>
-        from auth_param
-        where auth_id = #{authId,jdbcType=VARCHAR}
-        and app_name = #{appName,jdbcType=VARCHAR}
-        limit 1
+    <select id="findByAuthIdAndAppName" resultMap="BaseResultMap">
+        SELECT
+            <include refid="Base_Column_List"/>
+        FROM auth_param
```
Comment (midnight2104) [inline shenyu-admin/src/main/resources/mappers/auth-param-sqlmap.xml:60]: The keywords are not right-aligned.

File: shenyu-admin/src/main/resources/mappers/auth-param-sqlmap.xml [hunk 4]
```diff
@@ -28,100 +28,103 @@
...
-        app_param,
-        date_created,
-        date_updated)
+            (id,
+            auth_id,
+            app_name,
+            app_param,
+            date_created,
+            date_updated)
         VALUES
```
Comment (midnight2104) [inline shenyu-admin/src/main/resources/mappers/auth-param-sqlmap.xml:92]: The keywords are not right-aligned.

File: shenyu-admin/src/main/resources/mappers/auth-param-sqlmap.xml [hunk 5]
```diff
@@ -28,100 +28,103 @@
...
-        where id = #{id,jdbcType=VARCHAR}
+        DELETE FROM auth_param
+        WHERE id = #{id,jdbcType=VARCHAR}
     </delete>
 
     <delete id="deleteByAuthId" parameterType="java.lang.String">
-        delete from auth_param
-        where auth_id = #{authId,jdbcType=VARCHAR}
+        DELETE FROM auth_param
+        WHERE auth_id = #{authId, jdbcType=VARCHAR}
```
Comment (midnight2104) [inline shenyu-admin/src/main/resources/mappers/auth-param-sqlmap.xml:119]: The keywords `WHERE` are not right-aligned.

File: shenyu-admin/src/main/resources/mappers/auth-param-sqlmap.xml [hunk 6]
```diff
@@ -28,100 +28,103 @@
...
     </delete>
 
     <delete id="deleteByAuthIds" parameterType="java.util.List">
-        delete from auth_param
-        where auth_id in
-        <foreach item="authId" collection="list" open="(" separator="," close=")">
-            #{authId,jdbcType=VARCHAR}
-        </foreach>
+        DELETE FROM auth_param
+        WHERE auth_id IN
```
Comment (midnight2104) [inline shenyu-admin/src/main/resources/mappers/auth-param-sqlmap.xml:124]: The keywords `WHERE` are not right-aligned.

**Comment by hutaishi** [comment]
Thank you. I'll revise it

File: shenyu-admin/src/main/resources/mappers/auth-param-sqlmap.xml (hunk 1 above)
Comment (hutaishi) [reply shenyu-admin/src/main/resources/mappers/auth-param-sqlmap.xml]: Fixed

**Review by midnight2104 — APPROVED** [review]

**Pull request merged**